elastic/kibana
# [Stack Monitoring] functional tests - properly set test-subj attribute

## Summary

Fixes https://github.com/elastic/kibana/issues/114487

Nothing broke, just a couple of misplaced `test-subj` attributes.

### Testing
Disabled [closeAlertsModal](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/monitoring/cluster_overview.js#L78) and ran `./kibana/*` tests. [One test](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/apps/monitoring/kibana/instances_mb.js#L17) is currently skipped, I'll check if it can be enabled separately.

- set `render_react_app` to true https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/server/config.ts#L54
- ~comment out non-monitoring tests in [test config](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/config.js#L29) - should that be a glob argument ?~
- ~comment out non-kibana tests in [monitoring config](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/apps/monitoring/index.js#L30-L35)~
- start the test server from the kibana directory `node scripts/functional_tests_server --config x-pack/test/functional/config.js`
- run the tests `node scripts/functional_test_runner.js --config x-pack/test/functional/config.js --grep "Monitoring app Kibana"`

```
...
     └-> "after all" hook for "should have Instance Summary Status showing correct info"
   └-> "after all" hook in "Monitoring app"
5 passing (1.0m)
1 pending
````
